package.json

```diff
@@ -4,7 +4,7 @@
   "description": "An Application Shell for generating Angular Applications with Webpack and ES6",
   "main": "app.js",
   "scripts": {
-    "dev": "export NODE_ENV=development && webpack-dev-server --devtool=source-map --progress --colors --port=9000 --content-base app",
+    "dev": "export NODE_ENV=development && webpack-dev-server --progress --colors --port=9000 --hot --content-base app",
```

Comment:
Adding hot in both the script args and in `plugins` will break webpack. Remote --hot in a new commit.